CONTRIBUTING.md

```diff
@@ -116,7 +120,12 @@ In order to keep your fork up-to-date, you need to point it to the main SuperCol
 	- Checkout the `develop` branch: `git checkout develop`
 	- Pull changes into the `develop` branch: `git pull upstream develop`
 	- If you have already created your topic branch:
-		- Update it with the changes in the `develop`: `git rebase develop topic/branch-description`
+		- Update it with the changes in the `develop` branch using `rebase`: `git rebase develop topic/branch-description`
+		- **Note:** if you have already pushed your branch, rebasing will make your local branch diverge from the remote (`origin`). This may happen in the case that you need to make additional commits in response to developer feedback before your pull request is accepted. In that case you have two options:
```

Comment:
We should add: you don't usually need to rebase unless there have been new changes to relevant code or it's been a very long time. We shouldn't give the impression that this needs to be done every single time. Along the same lines, saying "First make sure your fork is up to date!" above is a bit zealous.